rocAL/rocAL/source/pipeline/ring_buffer.cpp

```diff
@@ -159,7 +151,7 @@ void RingBuffer::init(RocalMemType mem_type, void *devres, unsigned sub_buffer_s
 
         }
     }
-   else
+    else
```

Comment:
Please remove the additional space

Reply:
Added additional space to align with the if condition